core/templates/xregistry_openapi_template.json

```diff
@@ -1,1068 +1,1937 @@
+                        "required": false,
+                        "schema": { "type": "integer" }
+                    },
+                    {
+                        "in": "query",
+                        "name": "top",
+                        "description": "The number of resources to return (pagination).",
+                        "required": false,
+                        "schema": { "type": "integer" }
+                    },
+                    {
+                        "$ref": "#/components/parameters/filter"
+                    },
+                    {
+                        "$ref": "#/components/parameters/doc"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The collection of resources (optionally filtered).",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "additionalProperties": {
+                                        "$ref": "{%-resourceTypeReference-%}"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "400": {
+                        "$ref": "#/components/responses/BadRequest"
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/Unauthorized"
+                    },
+                    "404": {
+                        "$ref": "#/components/responses/NotFound"
+                    },
+                    "500": {
+                        "$ref": "#/components/responses/InternalServerError"
+                    }
                 }
-              }
             }
-          },
-          "404": {
-            "description": "No groups found"
-          }
-        }
-      }
-    },
-    "/{%-groupNamePlural-%}/{groupid}": {
-      "parameters": [
-        {
-          "$ref": "#/components/parameters/groupid"
-        }
-      ],
-      "get": {
-        "operationId": "get{%-groupNameSingular-%}",
-        "responses": {
-          "200": {
-            "description": "The resource group",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "{%-groupTypeReference-%}"
+        },
+        "/{%-groupNamePlural-%}/{groupid}/{%-resourceNamePlural-%}/{resourceid}": {
+            "parameters": [
+                {
+                    "$ref": "#/components/parameters/groupid"
+                },
+                {
+                    "$ref": "#/components/parameters/resourceid"
+                },
+                {
+                    "$ref": "#/components/parameters/specversion"
+                },
+                {
```

Comment:
openapi question... not being under an operation (e.g. 'get') does this mean it applies to ALL ops?
If so, we should probably do this at all levels of the tree